src/OpenTelemetry.AutoInstrumentation/Properties/AssemblyInfo.NetFramework.cs

```diff
@@ -18,9 +18,10 @@
 using System.Reflection;
 using System.Runtime.CompilerServices;
 
-[assembly: AssemblyKeyFileAttribute("keypair.snk")]
+[assembly: AssemblyKeyFile("keypair.snk")]
```

Comment:
Out of curiosity, why a change is needed here?

Reply:
Looks like an opportunistic refactor 😉

Reply:
Semi-auto fix by VS/R# recommendation. Not related.